src/App.js

```diff
@@ -15,15 +13,18 @@ export default class App extends Component {
 
   render() {
     return (
+      <Wrapper>
+        <GlobalStyle />
         <Router>
-              <Link to="/coins">Coins</Link>
-              <Link to="/portfolio">Portfolio</Link>
+          <NavBar/>
+          <MarketBar/>
           <Switch>
             <Route exact path="/" component={Coins} />
             <Route exact path="/coins" component={Coins} />
```

Comment:
you will need an id for this path

Reply:
I'll build this path after discussing with you. I have minor questions in the layout. I'm unsure if the ~~specific coin page~~ is inside coins or inside portfolio.